_config.yml

```diff
@@ -42,11 +42,13 @@ navigation:
   - title: About
     link: /about.html
   - title: People
-    link: /#people
+    link: /people
```

Comment:
Does this work for you locally ? The `#` indicates a heading, so it is telling the browser to stay within the original page. I'd expect it to fail when removed.

Reply:
It does. There's a "people" tab and a "people" section on the main page. The other tabs went to their tabs rather than jump down to the section. This just directs to the tab and makes it consistent with the other.

Reply:
Ah, ok. It's a bit of _tomato-tomahto_ since it directs users to the same place. I don't love that this is lumped in with the other changes, but not enough to block it.

Reply:
That's fair. Admittedly, I didn't realize it was in this. It's removed.